terraform/otel-ec2/vars.tf

```diff
@@ -51,3 +47,254 @@ variable "inventory_output" {
 variable "ansible_playbook" {
   default = "../../ansible/install-otelcol.yml"
 }
+
+variable "ec2_prefix" {
+  description = "EC2 instances names prefix, it will appended to the ec2_otels map key"
+  default = ""
+}
+
+variable "ec2_filters" {
+  description = "EC2 instances names to deploy"
+  default = []
+}
+
+variable "ec2_otels" {
```

Comment:
In the Infra Agent we have 44 AMIs. Why only 18 here? 👇

Reply:
Good point, I was focused on the SA as it is not a breaking change. I can add infra agent's ones too

Reply:
Sounds good. I'm finishing with packaging tests in the Infra Agent. Once is merged, I can add the rest.